src/tools/device-graph/matter-device-graph.py

```diff
@@ -0,0 +1,197 @@
+#
+#    Copyright (c) 2022 Project CHIP Authors
```

Comment:
```suggestion
#    Copyright (c) 2023 Project CHIP Authors
```

Reply:
Oh, the script I used as a base seems to be out-dated 😄 Thanks for catching!